.eslintrc.js

```diff
@@ -1,15 +1,52 @@
 module.exports = {
   root: true,
   parserOptions: {
-    ecmaVersion: 2017,
+    ecmaVersion: 2018,
     sourceType: 'module'
   },
+  plugins: [
+    'ember'
+  ],
+  extends: [
+    'eslint:recommended',
+    'plugin:ember/recommended'
+  ],
```

Comment:
Prettier is missing, also don't forget to run `yarn lint:js --fix` after adding it in the config..